extensions/ql-vscode/src/databases-ui.ts

```diff
@@ -297,10 +294,7 @@ export class DatabaseUI extends DisposableObject {
       commandRunnerWithProgress(
         "codeQLDatabases.chooseDatabaseGithub",
         async (progress: ProgressCallback, token: CancellationToken) => {
-          const credentials = isCanary()
```

Comment:
Note that this canary check has been moved into `promptImportGithubDatabase`, so there shouldn't be any behaviour change and we shouldn't be forcing non-canary users to log in.